agents/components/handoff-instructions.component.md

```diff
@@ -0,0 +1,30 @@
+````markdown
+## Handoff Mode and Stage Navigation
```

Comment:
This section should indicate that the user can ask the Status agent for help navigating the workflow - this is applicable to every stage. This should be states concisely as part of the handoff message.